### ISSUE TITLE
alwaysInjectSelector & neverInjectSelector are not indenting correctly in istio-sidecar-injector configmap  

### ISSUE THREAD

**Issue by softcane** [istio/istio#23027]
**Bug description**

Version: 1.5.1

alwaysInjectSelector & neverInjectSelector are not indented currently in the istio-sidecar-injector config map. istio-sidecar-injector and istiod related pods are crashing every time when we apply new manifest through istioctl. We have to manually patch the config map every time and rollout the istio-sidecar-injector and istiod deployment  

[X ] Installation

**Steps to reproduce the bug**
Create an istio manifest with istioctl and set the sidecarInjectorWebhook accordingly. 
Example:

```
  values:
    sidecarInjectorWebhook:
      neverInjectSelector:
        - matchExpressions:
            - key: test
              operator: DoesNotExist
```
**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

istioctl: 1.5.1
kubectl: v1.16.2

**How was Istio installed?**
Manifest generated from istioctl

**Comment by howardjohn** [comment]
This was fixed on master but not 1.5: https://github.com/istio/istio/pull/22738
cherrypick: https://github.com/istio/istio/pull/23033

**Comment by howardjohn** [comment]
Fixed ^
